apps/qbittorrent/5.0.2/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   qbittorrent:
-    image: linuxserver/qbittorrent:5.0.2
+    image: linuxserver/qbittorrent:5.0.3
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The code appears to be correct for updating the image version of Qbittorrent from "5.0.2" to "5.0.3". There are no irregularities or potential issues that I can detect in this change alone. If you have specific questions about running Docker containers or optimizing configurations, feel free to ask!